src/template_filters.rs

```diff
@@ -0,0 +1,260 @@
+use anyhow::Result;
```

Comment:
question: could we split this file into an own folder as a module `template_filter/` where we could have for every filter a very small own file?